bitcoin-rpc-provider/src/lib.rs

```diff
@@ -400,9 +439,12 @@ fn poll_for_fee_estimates(
         };
         match query_fee_estimate(&client, 6, EstimateMode::Conservative) {
             Ok(fee_rate) => {
-                fees.get(&ConfirmationTarget::HighPriority)
+                fees.get(&ConfirmationTarget::OnChainSweep)
                     .unwrap()
                     .store(fee_rate, Ordering::Release);
+                fees.get(&ConfirmationTarget::MaxAllowedNonAnchorChannelRemoteFee)
+                    .unwrap()
+                    .store(max(25 * 250, fee_rate * 10), Ordering::Release);
```

Comment:
In the link you provided on the right side it uses `HighPriority` which is 5000, just wondering why you're using the fee rate?

Reply:
Ah nevermind IIUC you got the fee estimate for `OnChainSweep` which is equivalent to `HighPriority` and use that.